### PR TITLE
fix: add AnuragThePathak

### LINKED ISSUE #874: Anurag Pathak
README

### PR DESCRIPTION
**Pull request by AnuragThePathak**
resolve #874

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
+  },
+  {
+  "name": "Anurag Pathak",
+  "githubUsername": "AnuragThePathak",
+  "imageUrl": "https://user-images.githubusercontent.com/75429250/143869337-38c7d5d6-f496-4435-9970-6d0831931884.png",
+  "issueNumber": 874
```
Comment (krishguptadev) [inline data.json:1662]:
```suggestion
    "issueNumber": 874
```

File: data.json [hunk 2]
```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
+  },
+  {
+  "name": "Anurag Pathak",
+  "githubUsername": "AnuragThePathak",
+  "imageUrl": "https://user-images.githubusercontent.com/75429250/143869337-38c7d5d6-f496-4435-9970-6d0831931884.png",
```
Comment (krishguptadev) [inline data.json:1661]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/75429250/143869337-38c7d5d6-f496-4435-9970-6d0831931884.png",
```

File: data.json [hunk 3]
```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
+  },
+  {
+  "name": "Anurag Pathak",
+  "githubUsername": "AnuragThePathak",
```
Comment (krishguptadev) [inline data.json:1660]:
```suggestion
    "githubUsername": "AnuragThePathak",
```

File: data.json [hunk 4]
```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
+  },
+  {
+  "name": "Anurag Pathak",
```
Comment (krishguptadev) [inline data.json:1659]:
```suggestion
    "name": "Anurag Pathak",
```

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by AnuragThePathak** [comment]
Do I need to commit and send pull request from main branch?

**Comment by krishguptadev** [comment]
> Do I need to commit and send pull request from main branch?

nope.

**Comment by AnuragThePathak** [comment]
> nope.

Then can you please tell me where did I go wrong?
I can see this only  ` Error: A branch or tag with the name 'AnuragPathak' could not be found `

**Comment by krishguptadev** [comment]
> > nope.
> 
> Then can you please tell me where did I go wrong? I can see this only `Error: A branch or tag with the name 'AnuragPathak' could not be found`

That is a known error, is being fixed upon 🙂

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**